test/testinput/acftbias_converter.yaml

```diff
@@ -0,0 +1,14 @@
+input coeff file: testinput/acftbias_in 
+default predictors: &default_preds
+- constant
```

Comment:
So this list should only be 3 instead of 9, 4-6, 7-9 should get appended from 1-3 in the code somehow.

Reply:
Probably need to ask @ADCollard about what these errors should be saved as.

Reply:
the background error ones should be stored as BiasCoeffErrors, not sure about the obs errors...